src/cpp/session/SessionOptions.cpp

```diff
@@ -274,7 +274,7 @@ core::ProgramStatus Options::read(int argc, char * const argv[])
          value<bool>(&autoReloadSource_)->default_value(false),
          "Reload R source if it changes during the session")
       ("r-compatible-graphics-engine-version",
-         value<int>(&rCompatibleGraphicsEngineVersion_)->default_value(11),
+         value<int>(&rCompatibleGraphicsEngineVersion_)->default_value(12),
```

Comment:
This all looks like the right approach to me. I'd just do some developer testing under the different versions and then have some QA testing to make sure nothing has gone amok with the device behavior (seems highly unlikely but worth some poking at).